software/bwamem2/index/meta.yml

```diff
@@ -0,0 +1,51 @@
+name: bwa_index
+description: Create BWA-mem2 index for reference genome
+keywords:
+    - index
+    - fasta
+    - genome
+    - reference
+tools:
+    - bwa:
+        description: |
+            BWA-mem2 is a software package for mapping DNA sequences against
+            a large reference genome, such as the human genome.
+        homepage: https://github.com/bwa-mem2/bwa-mem2
+        documentation: https://github.com/bwa-mem2/bwa-mem2#usage
+params:
+    - outdir:
+        type: string
+        description: |
+            The pipeline's output directory. By default, the module will
+            output files into `$params.outdir/<SOFTWARE>`
+    - publish_dir_mode:
+        type: string
+        description: |
+            Value for the Nextflow `publishDir` mode parameter.
+            Available: symlink, rellink, link, copy, copyNoFollow, move.
+    - enable_conda:
+        type: boolean
+        description: |
+            Run the module with Conda using the software specified
+            via the `conda` directive
+    - singularity_pull_docker_container:
+        type: boolean
+        description: |
+            Instead of directly downloading Singularity images for use with Singularity,
+            force the workflow to pull and convert Docker containers instead.
+input:
+    - fasta:
+        type: file
+        description: Input genome fasta file
+output:
+    - index:
+        type: file
+        description: BWA genome index files
+        pattern: "*.{0132,amb,ann,bwt.2bit.64,pac}"
+    - version:
+        type: file
+        description: File containing software version
+        pattern: "*.{version.txt}"
+authors:
+    - "@drpatelh"
```

Comment:
You wrote this! You can take my name out :)

Reply:
I only copied the bwa ones and replace bwa by bwa-mem2.
I feel you've done part of the work

Reply:
But I understand that you don't want to be responsible for something that you actually did not write